packages/flutter_tools/templates/plugin/pubspec.yaml.tmpl

```diff
@@ -6,7 +6,7 @@ homepage:
 
 environment:
   sdk: ">=2.7.0 <3.0.0"
```

Comment:
We could also change the sdk version, though it doesn't matter too much since it's shipped with Flutter.

Reply:
If you want to do that, it would become `>=2.9.0`

Reply:
I don't see any risk of that but since we are cherry picking this, let's make it as simple as possible. We can always update the sdk version later. WDYT?

Reply:
I don't have a preference.